### DISCUSSION TITLE
Castlevania ROM dump problem

### DISCUSSION THREAD

**Discussion by Gk2E7Aaojf3KDCQ** [sanni/cartreader#267] [Need help]
I have tried many, many times to get the first Castlevania to dump correctly, but I keep getting a different CRC value every time. I've more than thoroughly cleaned the contacts, but it just won't work. I've dumped dozens of other games without trouble.

What else can I try to make it work? I put the cartridge in the NES and it displays on the TV just fine. The game works, I just don't get why the PRG dump is different every single time. I think I have the Rev A version, as that's written on the back label and there are 3 screws, which the database I found says is indicative of Rev A.

**Comment by sanni** [comment]
Probably best to ask in the Arduino forum thread if somebody already successfully dumped Castlevania before to make sure it's not a bug in the Cart Reader code. https://forum.arduino.cc/t/rom-reader-for-super-nintendo-super-famicom-game-cartridges/155260/1418

**Comment by Aerik72** [comment]
Oh! Sorry to dig up such an old convo, but I have Castlevania, and dumped it successfully last night. I also have the 3-screw Rev-A, so it's not a problem with that game in particular.

I had a problem like you're describing with the Famicom cart of Final Fantasy III, but cleaning the cartridge contacts one last time with IPA and a cotton swab finally got it to read properly.

HTH
